docs/executor-cypress.md

```diff
@@ -1,14 +1,14 @@
 # Cypress Tests
 
-TestKube is able to make simple runs of Cypress tests. As Cypress is organised in projects we allow to define your tests in Github repository (for now only public one is implemented).
+TestKube makes running Cypress tests simple. As Cypress is organised in projects, tests can be defined in a Github repository (for now, only a public repository is implemented).
```

Comment:
We are now supporting private repos for cypress (and other executors). can you confirm ? @vsukhin @exu